schemas/backend-status-response.yml

```diff
@@ -0,0 +1,22 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:                      "Backend Status Response"
+description: |
+  Backend Status Response
+type:                       object
+properties:
+  status:
+    type: string
+    description: |
+      A string from Deadman's Snitch which describes the status.  See
+      https://deadmanssnitch.com/docs/api/v1#listing-your-snitches for an
+      explanation of this value
+  checked_in_at:
```

Comment:
Nit, but maybe don't use `_`, we don't do this anywhere...

I would suggest `updated` or something..

Regardless, you probably want `format: date-time`, then use `new Date().toJSON()` which is compatible.